packages/template-typescript-minimal/tsconfig.json

```diff
@@ -98,5 +98,6 @@
         /* Completeness */
         // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
         "skipLibCheck": true /* Skip type checking all .d.ts files. */
-    }
+    },
+    "exclude": ["node_modules", "app/application-extensions"]
```

Comment:
Another exclude/ignore that we'll address in one of the previously mentioned tickets.